docs/source/analyze/certificates.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Certificates
+description: Perform analysis and checks on Certificates found in the cluster
+---
+
+The Certificates analyzer serves to alert users when a specific certificate is either invalid or nearing its expiration date.
```

Comment:
Should "The Certificates analyzer" be "The `certificates` analyzer"? I noticed most analyzers/collectors in the troubleshoot docs use this backtick formatting

Reply:
Good catch, I have checked to use  backtick formatting